app.js

```diff
@@ -0,0 +1,16 @@
+var express = require("express"),
+	app 	= express();
```

Comment:
We shouldn't inline codes that are not in specific block. We can remove these spaces including below lines.